src/public/translations/en/translation.json

```diff
@@ -1656,5 +1656,27 @@
     "minutes": "Minutes",
     "hours": "Hours",
     "days": "Days"
+  },
+  "share": {
+    "title": "Share Settings",
+    "redirect_bare_domain": "Redirect bare domain to Share page",
+    "redirect_bare_domain_description": "When enabled, accessing the root URL will redirect to the Share page instead of Login",
+    "show_login_link": "Show Login link in Share theme",
+    "show_login_link_description": "Add a login link to the Share page footer"
+  },
+  "sync_2": {
```

Comment:
I think you accidentally copied the "sync_2" block here – I can find it twice in your 
https://github.com/TriliumNext/Notes/blob/657638ee54f93c5e278e2a65b68b98230de102a0/src/public/translations/en/translation.json

Reply:
doh! removed